docs/automation.md

```diff
@@ -87,4 +87,59 @@ This is a snapshot of our automated checks at a glance.
 
 ![Continuous Deployment practices](./media/continuous_deployment_practices.png)
 
+## Lambda layer build & deploy
+
+We fetch the latest PyPi release, then build Python **3.8-3.12** for **x86_64** and **arm64** architectures, using QEMU emulation for arm64. We create **a single CDK Asset** with x86_64 and arm for each Python version to optimize deployment performance.
```

Comment:
suggestion: break this up as you're assuming whoever reading knows the intricacies of all that.

> Questions in my head for the non-initiated (new maintainers)

* Why do we build a Layer for each Python version?
* Why do use QEMU for emulating ARM64?
* What's a CDK Asset? can we use a different name?

You could also use sub-heading to help split context and how it's done

Reply:
Good catch @heitorlessa! I think using sub-heading make this too extensive, but I added some bullet points. I also removed QMU because this doesn't bring any value in this section.

Can you pls review?